app/webserver/webserver.go

```diff
@@ -146,11 +144,11 @@ func wsReader(conn *websocket.Conn) {
 				log.Errorln("[axolotl] Open chat with id: ", openChatMessage.Id, "failed", err)
 			} else {
 				activeChat = openChatMessage.Id
-				if !s.IsGroup {
-					// TODO: Avatar and profile handling for private chats, decryption is not yet done on the textsecure part
-					// p, _ := textsecure.GetProfile(s.Tel)
-					// profile = p
-				}
+				// TODO: Avatar and profile handling for private chats, decryption is not yet done on the textsecure part
```

Comment:
Is there any issue or PR for this somewhere on the side of Signal?
Just to have some further info when this could be brought back

Reply:
The profile avatar decryption is not yet done and the profile is not yet used in axolotl web. Further the content of the if was commented.
To get that right we need this https://github.com/signal-golang/textsecure/pull/22 and this resolved https://github.com/nanu-c/axolotl/issues/8